app/components/Popover.tsx

```diff
@@ -1,40 +1,50 @@
 import * as React from "react";
 import { Dialog } from "reakit/Dialog";
-import { Popover as ReakitPopover } from "reakit/Popover";
+import { Popover as ReakitPopover, PopoverProps } from "reakit/Popover";
 import styled from "styled-components";
 import breakpoint from "styled-components-breakpoint";
 import useMobile from "~/hooks/useMobile";
 import { fadeAndScaleIn } from "~/styles/animations";
 
-type Props = {
+type Props = PopoverProps & {
+  children: React.ReactNode;
+  // TODO what is tabindex for
```

Comment:
@tommoor question for you — this seems odd. It looks like tabindex is getting trapped here and not passed down to children on purpose?

Reply:
It's being passed in the `...rest` no?

Reply:
yeah I got confused >_<

My original question was because I took away the typing and tsc doesn't complain. It seems unnecessary to me, but leaving it in as it's harmless for now